dependencies/docker/zsh-vendor-completions.zsh

```diff
@@ -1,3126 +0,0 @@
-#compdef docker dockerd
-#
-# zsh completion for docker (http://docker.com)
```

Comment:
This file provided in this repo for https://github.com/kachick/dotfiles/issues/236, I don't want to put as this if podman does not make similar error.

Reply:
I was wrong https://github.com/kachick/dotfiles/pull/423